api/src/Modules/staff/App/Routes.php

```diff
@@ -28,6 +28,7 @@ function () use ($app, $authMiddleware) {
         function () use ($app, $authMiddleware) {
             $app->get('/staff/', 'App\Modules\staff\Controller\ListStaff');
             $app->get('/{id}/staff', 'App\Modules\staff\Controller\GetDepartment');
+            $app->get('/staff/position', 'App\Modules\staff\Controller\ListDepartmentPosition');
```

Comment:
I can rename the path to be plural, no problem.

I agree, there didn't seem to be a great place to put this. Originally I was thinking it might make sense to include under `/staff` but I saw that we already had `/staff_membership` and it didn't seem quite right there.

How would you feel about `/staff`? For positions it would be `/staff/positions`

As a side note, are the endpoints under `/staff_membership` still planning to be used? It seems like the GET endpoint is only returning the first member that appears for a given position ID and the DELETE endpoint is only deleting the first member that appears for a given position ID. I know these were a work in progress, but just wondering if they still make sense.

If so, and we go with `/staff` for the position list, perhaps we can adjust these endpoints to also be under `/staff` grouping as well?

Reply:
I am up for moving `staff_membership` to `staff/membership` then you have `staff` as a base that would work great for `staff/positions`  Then you can also map the `POST` and `PUT` methods when you get there into `staff/position`